config.json

```diff
@@ -26,6 +28,7 @@
   { "target": "ipfs/go-fetcher" },
   { "target": "ipfs/go-filestore" },
   { "target": "ipfs/go-fs-lock" },
+  { "target": "ipfs/go-hamt-ipld" },
```

Comment:
Hm. Actually, I'm going to archive this repo.

Reply:
Great. A test for our cron workflow!

Reply:
Ok, closing https://github.com/protocol/.github/pull/151.